CHANGELOG/CHANGELOG-1.2.md

```diff
@@ -38,6 +48,13 @@ We are working on bringing Azure Portal support for the new property soon. Until
 * Switch from `hostname` to `hostnames` property in HTTP Listener on Application Gateway.
 * helm: remove replica count setting from supported helm values. This will be added back when AGIC will recieve support for leader election.
 
+## Known Issues:
```

Comment:
In addition, we have a fix in rc3 to address prohibited target with hostnames, which means rc2 doesn't have the fix...better to notify here in case rc2 is used for shared feature?